src/picosvg/svg_meta.py

```diff
@@ -95,25 +95,28 @@ def cmd_coords(cmd):
     return _CMD_COORDS[cmd]
 
 
-def ntos(n):
-    # %f likes to add unnecessary 0's, %g isn't consistent about # decimals
-    return ("%.3f" % n).rstrip("0").rstrip(".")
+def ntos(n: float) -> str:
+    # strip superflous .0 decimals
+    return str(int(n)) if isinstance(n, float) and n.is_integer() else str(n)
```

Comment:
Curiosity question, wouldn't it work to just remove the .3 from the original code? - all else being equal I prefer a smaller diff